core/miner/storage_fsm/sealing_events.hpp

```diff
@@ -10,6 +10,8 @@
 #include "miner/storage_fsm/types.hpp"
 
 namespace fc::mining {
+  using api::SectorNumber;
```

Comment:
remove

Reply:
then there will be a build fault

Reply:
It means that you need to declare it somewhere else where it is used. Not here